### DISCUSSION TITLE
Does notebook have an independent setting of word wrap?

### DISCUSSION THREAD

**Discussion by yuuuxt** [microsoft/vscode-jupyter#11199] [Questions and Answers] [answered]
I only found `"editor.wordWrap": "on"` in settings. 

I want to set word wrap to `off`  for notebook, while keeping `"editor.wordWrap": "on"` for all other situations.

(The reason is that there are some repetitive long expressions, which look better without word wrap.)

**Comment by rebornix** [comment, marked as answer]
We can override editor settings in notebook only through setting `notebook.editorOptionsCustomizations`. For example, to disable word wrap for notebooks:

```json
{
  "editor.wordWrap": "on"
  "notebook.editorOptionsCustomizations": {
    "editor.wordWrap": "off"
  }
}
```
